package.json

```diff
@@ -32,6 +32,9 @@
     "mongoose": "^8.2.4",
     "node": "^18.20.0",
     "socket.io": "^4.7.5",
+    "swagger": "^0.7.5",
```

Comment:
This package add 35 vulnerabilities. I do not see where it is used it should be removed